.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,7 +4,29 @@ Please write a summary of the changes, and please include relevant motivation an
 
 # Result
 
-If you've made visual changes, please include **before** and **after** images, preferably with a description. Make sure they do not contain any real user information.
+If you've made visual changes, please check the boxes below and include images showing the changes. Descriptions are appreciated.
+
+- [ ] Changes look good on both light and dark theme.
+- [ ] Changes look good with different viewports (mobile, tablet, etc.).
+- [ ] Changes look good with different Internet connections.
+
+> [!CAUTION]
+> Make sure your images do not contain any real user information.
+
+<table>
+	<tr>
+		<td>Before</td>
+		<td>After</td>
+	</tr>
+	<tr>
+		<td>
+			...
+		</td>
+		<td>
+			...
+		</td>
```

Comment:
Personally fan of having three columns, so you describe each row in the first - but if there are other preferences out there lets hear it

Reply:
I'm just more used to two so that there's more space for the images, but three is fine by me

Reply:
Added a third column now, but limited it to a width of 25%